pubspec.yaml

```diff
@@ -14,7 +14,7 @@ description: A new Flutter project.
 version: 1.0.0+1
 
```

Comment:
Why did we change the SDK version, if someone changes it then we all have to make that change?